src/blazar_tempest_plugin/tests/scenario/test_oshost_basic_ops.py

```diff
@@ -32,9 +32,8 @@ class TestReservableBaremetalNode(ReservationScenarioTest):
     @classmethod
     def skip_checks(cls):
         super().skip_checks()
-        if not CONF.service_available.ironic:
-            skip_msg = "Ironic service is not available"
-            raise cls.skipException(skip_msg)
+        if not CONF.reservation_feature_enabled.host_plugin:
+            raise cls.skipException("host reservations are not enabled")
```

Comment:
the service_available.ironic config toggle is only present if the ironic-tempest-plugin is installed
additionally, there may be cases where blazar host reservation isn't enabled, but ironic is.